### PR TITLE
The great floor tile purge

### PR DESCRIPTION
**Pull request by WJohn**
Basically:

This turns all of these into floor decals, much like we did with warning lines. This gives us far more flexibility when mapping, letting us make any combination of colors and no longer be restricted by airless tiles having pretty much no colored tile support. This also largely cuts down on the hugely bloated floors.dmi to make it a bit more manageable. 

White colored floor tiles may appear as too bright initially, but I have some planned changes to floor tiles in general in the relatively near future. Besides that, everything should look mostly the same. 

:cl: 
refactor: Refactored floor tiles to use decals like warning lines do.
/:cl:

### CODE REVIEW

**Comment by 81Denton** [comment]
Mama mia, that's amazing.

**Comment by kingoftheautism** [comment]
when this is going to be merged i will have to do something about my wip map. oof.

cool pr though

**Comment by yoyobatty** [comment]
Doesn't crow-barring floors remove the decal?

**Comment by Mark9013100** [comment]
@yoyobatty It does.

**Comment by 81Denton** [comment]
This is going to be amazing for mapping. Only downside I can see is what @yoyobatty mentioned.

**Comment by Ratvar** [comment]
Should goonplague be it's own floortile at all?

**Comment by WJohn** [comment]
Goonplaque should probably be a decal, yes. I'll do that and a number of other things to this file in the future.

**Comment by SpaceManiac** [comment]
Some of your replacements are of the pattern `circuit/killroom` -> `circuit/telecomms`.

These have different atmos mixes:

```
/turf/open/floor/circuit/killroom
	name = "killroom floor"
	initial_gas_mix = "n2=500;TEMP=80"

/turf/open/floor/circuit/telecomms
	initial_gas_mix = "n2=100;TEMP=80"
```

Is this intentional?

By the way, one of the reasons goonplaque is not yet a decal is it has `name` and `desc` set, because it's supposed to be examinable, to read the plaque. It's not a pure visual.

**Comment by ExcessiveUseOfCobblestone** [comment]
you should type those as `.../circuit/cold` tbh

**Comment by vuonojenmustaturska** [comment]
i put this on bagil as a testmerge

**Comment by Mark9013100** [comment]
@WJohn Could try asking Zuhayr from Bay, he, IIRC, did this two years ago.

**Comment by BeeSting12** [comment]
How will this affect current maps?

**Comment by WJohn** [comment]
Visually there'll be very little difference. As far as mapping is concerned, you just use these decals to cover these tiles like you would with warning lines.

**Comment by WJohn** [comment]
There we go, hideous eye cancer from transparency not being supported is fixed. Thanks to spacemaniac and ninja for helping me with this because I am fucking awful at the most basic of coding.

File: _maps/map_files/Deltastation/DeltaStation2.dmm [hunk 1]
```diff
@@ -10619,6 +10615,10 @@
 /obj/effect/turf_decal/tile/neutral{
 	dir = 8
 	},
+/mob/living/simple_animal/hostile/lizard{
+	name = "Eats-The-Roaches";
+	real_name = "Wags-His-Tail"
```
Comment (octareenroon91) [inline _maps/map_files/Deltastation/DeltaStation2.dmm]: Could you tidy up this varedit while you're changing things?

**Comment by Poojawa** [comment]
@WJohn Is there any special deployment of your script required?

**Comment by miniusAreas** [comment]
@WJohn You left out the stageedge tiles, and I would love to have half tiled half plated tiles or decals for adding plating to tiled tiles.

**Comment by ExcessiveUseOfCobblestone** [comment]
Wjohn doesn't code much here anymore

**Pull request merged**